OIPA/requirements.txt

```diff
@@ -1,5 +1,5 @@
-lxml==3.4.0
```

Comment:
Great, that means we can rip out the installment of headers in https://github.com/catalpainternational/ansible/blob/master/host_vars/mohinga_beta.yaml#L8-L9